modules/API/external/external_api.js

```diff
@@ -366,6 +366,23 @@ export default class JitsiMeetExternalAPI extends EventEmitter {
         return iframe.contentWindow.document.getElementById('largeVideo');
     }
 
+    /**
+     * Getter for participant specific video element in Jitsi Meet.
+     * 
+     * @param {string} participantId - Participant id
+     * or "local" for local video.
+     * @returns {HTMLElement|undefined} - The requested video.
+     */
+    _getLargeVideo(participantId) {
```

Comment:
Another ESlint error - Duplicate name '_getLargeVideo' eslint(no-dupe-class-members)

Why not rename the method to something like `_getParticipantVideo`?  I don't think it has something to do with the large video anyway.